### PR TITLE
Initialize components

### PR DESCRIPTION
**Pull request by carloso0114**
In this feature, I was able to recreate the wireframe by building components to look alike it's using real information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,44 +1,69 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app), using the [Redux](https://redux.js.org/) and [Redux Toolkit](https://redux-toolkit.js.org/) template.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Math Magicians
+
```
Comment (Emmyn5600) [inline README.md:4]: - [ ] This is the book store project not Math Magicians project, kindly correct this typo.

File: src/components/addBookForm.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+          {/* <option value="Category" selected disabled hidden>Category</option> */}
+          <option defaultValue="" hidden>Category</option>
+          <option value="1">Comedy</option>
+          <option value="2">Drama</option>
+          <option value="3">Fiction</option>
+        </select>
+        <div className="rectangle-4">
+          <span className="add-book">
+            ADD BOOK
+          </span>
```
Comment (Emmyn5600) [inline src/components/addBookForm.js:20]: - [ ] CreateNewBook
Has a button to submit great job
the button wouldn't add the book yet, this is just a placeholder for now, great work. However, it has to be a button not a text, make it as pointer.
react3

File: src/redux/books/books.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        <div>
+          <span className="genre">Action</span>
+          <h5 className="title">The Hunger Games</h5>
+          <span className="author">Suzanne Collins</span>
+        </div>
+        <div className="actions d-flex align-items-center">
+          <span>Comments</span>
+          <div className="Line-2" />
+          <span>Remove</span>
+          <div className="Line-2" />
```
Comment (Emmyn5600) [inline src/redux/books/books.js:18]: - [ ] RemoveBook
Has a button for removing books
the button wouldn't remove the books yet, this is just a placeholder for now, great work. However, it has to be a button not a text, make it as pointer.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @carloso0114 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
   - The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Categories still under construction:heavy_check_mark:

Changes required :writing_hand:  

      * #### Please check the comments under the review.
       
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**